src/tasks/AutoHeistTask.py

```
@@ -149,9 +162,10 @@ def __init__(self, *args, **kwargs):
         self.paths = {
             "路径1(路线参考自B站UP: 早柚大魔王丶)": HeistPathA,
             "路径2(在路径1基础上优化了大厅到办公层的路线)": HeistPathB,
+            "路径3(使用残虹避战，更加安全)": HeistPathC,
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)